### ISSUE TITLE
TODO for decoding in set_current_page_encoding() method

### ISSUE THREAD

**Issue by sadikovi** [sunchao/parquet-rs#51]
I encountered this TODO the other day, and thought if we should fix it, since we added support for pretty much all decodings in Parquet.

Line with TODO: https://github.com/sunchao/parquet-rs/blob/master/src/column/reader.rs#L315

Suggest we open PR to add it. I am not sure how we are going to test - adding files with new encodings or using make pages function?

**Comment by sadikovi** [comment]
@sunchao Could you comment on this TODO? Should we address it now? It does not sound particularly complicated, just need to figure out how to test it.

**Comment by sunchao** [comment]
@sadikovi Yes, good point. We can address this now. It should be easily turned on, and most of the effort will be on testing.
